static/components/PoiCard.js

```diff
@@ -9,8 +9,10 @@ class PoiCard extends React.Component {
     super(props)
     this.state = {
       from: {},
-      to: {}
+      to: {},
     }
+    this.hoverCard = this.hoverCard.bind(this)
```

Comment:
It's from react doc ... https://facebook.github.io/react/docs/handling-events.html